addons/mail/static/tests/message/message_tests.js

```diff
@@ -12,6 +12,17 @@ import { click, contains, insertText } from "@web/../tests/utils";
 
 const { DateTime } = luxon;
 
+const SELECTORS = {
+    // Actions
+    expand: ".o-mail-Message [title='Expand']",
+    reply: ".o-mail-Message [title='Reply']",
+    markAsTodo: ".o-mail-Message [title='Mark as Todo']",
+    // Expanded Actions
+    edit: ".o-mail-Message-actions [title='Edit']",
+    delete: ".o-mail-Message-actions [title='Delete']",
+    markAsUnread: ".o-mail-Message-actions [title='Mark as Unread']",
+};
```

Comment:
Please don't make indirection like that, it makes the test much harder to read/maintain. It's just as easy doing a replace all.

Reply:
Indeed it was not really necessary to make this change, fixed it!